module/module.go

```diff
@@ -543,17 +532,7 @@ func (m *Module) ReconfigureResource(ctx context.Context, req *pb.ReconfigureRes
 		m.logger.Error(err)
 	}
 
-	// check if our streams have been removed by reconfiguring and remove them if so
-	prs, ok := m.activeResourceStreams[res.Name()]
-	if ok {
-		m.logger.CInfow(ctx, "Cleaning up orphaned activeResourceStreams for resource: ", "name", res.Name())
-		if err := m.pc.RemoveTrack(prs.sender); err != nil {
-			m.logger.Error(err)
-		}
-
-		delete(m.activeResourceStreams, res.Name())
-	}
-
```

Comment:
This work is now done by `Close()` which causes the `Subscrption.Context` of all subscriptions to be cancelled, which then run the cleanup.

Reply:
Which `Close` is this? `res[ource].Close`?

Reply:
yes, resource.Close()